### PR TITLE
add buddy board support

### PR DESCRIPTION
**Pull request by adambujak**

### CODE REVIEW

File: firmware/applications/SignBuddy/nucleo_board.h [hunk 1]
```diff
@@ -21,9 +21,9 @@
 #define SYSCLK_FREQ               24000000
 
 /* LED */
-#define LED1_PORT                 GPIOC
-#define LED1_PIN                  LL_GPIO_PIN_13
-#define GPIO_LED_CLK_EN()         LL_IOP_GRP1_EnableClock(LL_IOP_GRP1_PERIPH_GPIOC)
+#define LED1_PORT                 GPIOA
+#define LED1_PIN                  LL_GPIO_PIN_8
+#define GPIO_LED_CLK_EN()         LL_IOP_GRP1_EnableClock(LL_IOP_GRP1_PERIPH_GPIOA)
```
Comment (stephen99scott) [inline firmware/applications/SignBuddy/nucleo_board.h:26]: PA5

**Review by stephen99scott — APPROVED** [review]

**Comment by stephen99scott** [comment]
Closes #139

File: firmware/applications/SignBuddy/buddy_board.h [hunk 2]
```diff
@@ -0,0 +1,159 @@
+#ifndef BUDDY_BOARD_H
+#define BUDDY_BOARD_H
+
+#error buddy
```
Comment (adambujak) [inline firmware/applications/SignBuddy/buddy_board.h:4]: must remove

File: firmware/applications/SignBuddy/nucleo_board.h (hunk 1 above)
Comment (adambujak) [reply firmware/applications/SignBuddy/nucleo_board.h:26]: how do we handle what is already on pa5?

Comment (stephen99scott) [reply firmware/applications/SignBuddy/nucleo_board.h:26]: ah forget about that

Comment (stephen99scott) [reply firmware/applications/SignBuddy/nucleo_board.h:26]: maybe for the nucleo board we can disable some of the touch sensors

**Pull request merged**